metabase/metabase
# Migrate ForgotPassword form to formik

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- `npx maildev -s 25 -w 80` to run a dev email server
- Admin -> Settings -> Email and setup email according to these values https://github.com/metabase/metabase/blob/c0e14e3364239625de924348fd775e89670e2995/frontend/test/__support__/e2e/helpers/e2e-email-helpers.js#L10
- Logout
- Login -> Forgot password
- It should be possible to enter your email and reset your password

<img width="549" alt="Screenshot 2022-11-01 at 20 07 41" src="https://user-images.githubusercontent.com/8542534/199306454-246ef0d5-4e49-40ba-953f-f695b51478dd.png">

<img width="525" alt="Screenshot 2022-11-01 at 20 08 02" src="https://user-images.githubusercontent.com/8542534/199306430-0edf2507-79d0-4c9d-99df-9ec7a19890c6.png">
